tests/DynamoDbModelTest.php

```diff
@@ -249,6 +280,9 @@ protected function seed($attributes = [])
             'name' => ['S' => str_random(36)],
             'description' => ['S' => str_random(256)],
             'count' => ['N' => rand()],
+            'version' => ['N' => '1'],
```

Comment:
Do we have a test for when no version or timestamps are enabled? for backward compatibility